src/app/shared/services/cluster.service.ts

```diff
@@ -93,6 +93,11 @@ export class ClusterService {
     this._cluster.labels = labels;
   }
 
+  set podNodeSelectorAdmissionPluginConfig(config: object) {
+    delete this._cluster.spec.podNodeSelectorAdmissionPluginConfig;
```

Comment:
Is this line required? In the next one we are overwriting the same property.

Reply:
you're right. removed it.